spark_log_parser/parsing_models/exceptions.py

```diff
@@ -20,84 +21,106 @@ def __init__(
         self.status_code = status_code
 
         # 2022-03-25 RW:  Format the information in the way it is expected by the backend code
-        self.error = {
-            "error"   : error_type,
-            "message" : error_message
-        }
-
-        logger.error(error_message)
-        if exception:
-            logger.exception(exception)
-
+        self.error = {"error": error_type, "message": error_message}
 
     def get_ui_return_value(self) -> dict:
         """
         A possible rendering of one set of return information as dict
         """
-        return {
-            "error": self.error_type,
-            "message": self.error_message
-        }
+        return {"error": self.error_type, "message": self.error_message}
 
     def get_ui_return_value_as_json(self) -> json:
         """
         A possible rendering of one set of return information as JSON
         """
-        return json.dumps( {
-            "error"   : self.error_type,
-            "message" : self.error_message
-        } )
+        return json.dumps({"error": self.error_type, "message": self.error_message})
 
-class ConfigurationException(SyncParserException):
+    def get_exception_response(self) -> dict:
+        """
+        Used to get an example response, which will be used for
+        the swagger documentation.

```

Comment:
I think it better to keep the log parser and prediction models agnostic about the context in which they are used. Specifically, they shouldn't be concerned with the design of a web API.

Reply:
Also, are you suggesting to delete just this comment, or the method entirely?